server/src/main/scala/com/faleknatalia/cinemaBookingSystem/Main.scala

```diff
@@ -23,9 +20,8 @@ object Main extends JsonSupport {
     implicit val system = ActorSystem(Behaviors.empty, "my-system")
     // needed for the future flatMap/onComplete in the end
     implicit val executionContext = system.executionContext
-    val movieService = new MovieService()
     val db = Database.forConfig("db")
-    val movies = TableQuery[MovieTable]
+    val movieService = new MovieService(db)(executionContext)
     val scheduledMovies = TableQuery[ScheduledMovieTable]
```

Comment:
`val scheduledMovies = TableQuery[ScheduledMovieTable]` powinno byc przeniesione do serwisu

Reply:
Z ksiazki o Slicku: 
"As a rule, use lazy val for TableQuery s and def foreign keys (for consistency
with column definitons)."